src/components/input/example-input-search-with-character-check.njk

```diff
@@ -12,12 +11,12 @@
         },
         "width": "10",
         "charCheckLimit": {
-            "charcheckCountdown": true,
-            "limit": 10,
-            "charCountSingular": "{x} more character needed",
-            "charCountPlural": "{x} more characters needed",
-            "charCountOverLimitSingular": "{x} character too many",
-            "charCountOverLimitPlural": "{x} characters too many"
```

Comment:
The ending bracket needs fixing on line 21 but these need reverting

Reply:
reverted :)